src/consumer/consumer.js

```diff
@@ -0,0 +1,54 @@
+import Frame from './frame';
+
+/**
+ * Add the given event handler to an EventEmitter instance
+ * @param {EventEmitter} emitter - an instance of EventEmitter
+ * @param {string} event - event name
+ * @param {function} handler - event handler function
+ */
+function addEventHandler(emitter, event, handler) {
+  if (typeof handler === 'function') {
+    emitter.on(event, handler);
```

Comment:
i would just move this code inline

Reply:
Updated. No more functions outside of class.